app/models/course_coverage.rb

```diff
@@ -1,6 +1,6 @@
 class CourseCoverage < SimpleDelegator
   def has_coverages?
-    coverages.present?
+    coverages.where(archived: false).present?
```

Comment:
Sadly, the `[courses_controller](https://github.com/MIT-IR/abet/blob/master/app/controllers/manage_assessments/courses_controller.rb)` depends on the `has_coverages` method and without specifying `archived: false`, I ran into problems.